modules/cram/merge_gvcf.nf

```diff
@@ -0,0 +1,18 @@
+include { basename } from './utils'
+
+// TODO --bed seems to be broken: https://github.com/dnanexus-rnd/GLnexus/issues/279
+process merge_gvcf {
+  input:
+    tuple val(key), val(meta), path(gVcfs), path(gVcfIndexes)
+  output:
+    tuple val(key), val(meta), path(vcfOut), path(vcfOutIndex), path(vcfOutStats)
+  shell:
+    basename = basename(meta[0])
+    vcfOut = "${basename}_merged.vcf.gz"
+    vcfOutIndex = "${vcfOut}.csi"
+    vcfOutStats = "${vcfOut}.stats"
+
+    config=params.vcf.gvcf_merge_preset
```

Comment:
it doesn't feel right that a `cram` workflow process depends on a `vcf` param. how about moving the param up one level to `nxf.config` and renaming it to `params.common.gvcf_merge_preset'?